.github/workflows/ci-typescript.yml

```
@@ -40,7 +40,8 @@ jobs:
           cache: 'pnpm'
       - run: pnpm install
       - run: pushd modules && pnpm test; popd
```

> Only run the tests etc for the current subproject in the matrix (and the cdk project, which we always need)

we will also run all the tests for all the modules

---

👍  Yeah - I think that's okay. Maybe as a future alternative we could run the module tests in a separate step and fail early if they're broken.

---

@johnduffell I've changed this in #2359 where I've introduced a separate job for the common stuff (module tests, module formatting)